Tools/AP_Periph/can.cpp

```diff
@@ -977,79 +994,112 @@ static bool shouldAcceptTransfer(const CanardInstance* ins,
     return false;
 }
 
+static void cleanup_stale_trx(uint64_t &timestamp_usec)
```

Comment:
what is trx? If I'm having to ask, perhaps we need a better name..

Reply:
transactions

Reply:
Rename it to that, please